utils/convert.py

```diff
@@ -1,60 +1,116 @@
-#!/usr/bin/python
-
 from __future__ import division
-from colormath.color_objects import LabColor, sRGBColor
+from colormath.color_objects import LabColor, sRGBColor, AppleRGBColor, BaseRGBColor, HSVColor
 from colormath.color_conversions import convert_color
-from pprint import pprint as pp
 import re
 import sys
 
-USAGE = """Convert from sRGB to Lab or from Lab to sRGB.
+USAGE = """Convert color(s) to all color spaces used by Selenized.
 
-The argument should either be a color (e.g. #c4d8df for sRGB, 23,-8,15 for Lab)
-or a path to a file with one color per line (empty lines allowed)."""
+The argument can be:
+  - a string representation of a color (e.g. c4d8df for sRGB, 23,-8,15 for Lab)
+  - a path to a file with one color per line (empty lines allowed)"""
 
-if len(sys.argv) != 2:
-    print USAGE
-    sys.exit()
 
-hex_num = r'([0-9a-fA-F]{2})'
-dec_num = r'(-?[0-9]{1,3})'
-hex_color_re = re.compile(r'#?' + hex_num*3)
-dec_color_re = re.compile(',?\s*'.join([dec_num]*3))
+# Define some pretty printing for classes from colormath module.
+def lab_str(self):
+    return "{:>3.0f} {:>3.0f} {:>3.0f}".format(
+        self.lab_l, self.lab_a, self.lab_b
+    )
 
-def pprint_lab(color):
+def hsv_str(self):
     return "{:>3.0f} {:>3.0f} {:>3.0f}".format(
-        color.lab_l,
-        color.lab_a,
-        color.lab_b
+        self.hsv_h, self.hsv_s*100, self.hsv_v*100
     )
 
-def parse_color(color):
-    coords = hex_color_re.match(color)
+# def rgb_str(self):
+#     return "{:>3.0f},{:>3.0f},{:>3.0f}".format(
+#         self.rgb_r*255, self.rgb_g*255, self.rgb_b*255
+#     )
+
+LabColor.__str__      = lab_str
+HSVColor.__str__      = hsv_str
+sRGBColor.__str__     = sRGBColor.get_rgb_hex
+AppleRGBColor.__str__ = AppleRGBColor.get_rgb_hex
```

Comment:
Ouch, how ugly (modyfing classes on the fly). Ouch, how pythonic (modyfing classes on the fly).

About formatting: I'm rather against such alignment of assignments, it ruins statements own readability for questionable esthetic reasons.